### PR TITLE
numa_memory_migrate: Fix the list error

### PR DESCRIPTION
**Pull request by hsmj1412**
Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/numa/numa_memory_migrate.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def mem_compare(used_node, left_node):
     # Get host numa node list
     host_numa_node = utils_misc.NumaInfo()
     node_list = host_numa_node.online_nodes_withmem
+    total_online_node_list = host_numa_node.online_nodes
```
Comment (balamuruhans) [inline libvirt/tests/src/numa/numa_memory_migrate.py]: It would be better to use online_nodes_withmem than online_nodes, how does this fix list error

Comment (hsmj1412) [reply libvirt/tests/src/numa/numa_memory_migrate.py]: @balamuruhans we have used online_nodes_withmem in line 54, but in line 144, the get_numa_status will report all of the nodes status. e.g. there are 4 nodes, [0,1,4,5]. Only node 0 and node 4 could be used(with memory). So, online_nodes_withmem will return[0,4], but get_numa_status will return the memory list [100,0,100,0]

**Comment by hsmj1412** [comment]
@chunfuwen Updated

File: libvirt/tests/src/numa/numa_memory_migrate.py (hunk 1 above)
Comment (balamuruhans) [reply libvirt/tests/src/numa/numa_memory_migrate.py]: @hsmj1412 Thanks for the explanation, fix looks good to me.

**Comment by balamuruhans** [comment]
ACK

**Comment by balamuruhans** [comment]
commit msg have a typo, %s/memroy/memory.

**Comment by hsmj1412** [comment]
@balamuruhans Fixed

**Pull request merged**